src/parser/jobs/mch/modules/GeneralCDDowntime.tsx

```diff
@@ -0,0 +1,47 @@
+import {Trans} from '@lingui/react'
+import ACTIONS from 'data/ACTIONS'
+import {CooldownDowntime} from 'parser/core/modules/CooldownDowntime'
+import React from 'react'
+
+// Strict downtime allowance for drill/bio blaster/air anchor
+const DOWNTIME_ALLOWED_GCD = 100
+
+export default class GeneralCDDowntime extends CooldownDowntime {
+	trackedCds = [ {
+		cooldowns: [ACTIONS.WILDFIRE],
+		firstUseOffset: 10000,
+	}, {
+		cooldowns: [ACTIONS.BARREL_STABILIZER],
+		firstUseOffset: 3000,
+	}, {
+		cooldowns: [ACTIONS.REASSEMBLE],
+		allowedAverageDowntime: 5000,
+		firstUseOffset: 0,
+	}, {
+		cooldowns: [ACTIONS.AIR_ANCHOR],
+		allowedAverageDowntime: DOWNTIME_ALLOWED_GCD,
+		firstUseOffset: 9000,
+		// isAffectedBySpeed: true,
```

Comment:
I'm not actually familiar with this flag; is there a reason it's here but commented out? From the name it seems like it should be uncommented (and if we don't actually need it, it would probably be cleaner to just delete it entirely). Same thing for the Drill/Bioblaster blob below.

Reply:
Following convention from the SMN module here, this flag would indeed be useful but it seems like it's been disabled since GCD estimates within xiva aren't very accurate (yet).